lib/chartmogul/api/actions/all.rb

```diff
@@ -11,6 +11,7 @@ def self.included(base)
         module ClassMethods
           def all(options = {})
             resp = handling_errors do
+              puts resource_path.apply_with_get_params(options)
```

Comment:
Is this puts supposed to be here?

Reply:
Oh, no, I don't even remember why I did this.